using-mobile-extensions/adobe-target-vec.md

```diff
@@ -207,3 +206,39 @@ You can disable the Target VEC extension to make the network request automatical
 
 You can set the `at_property` value for your workspace using the Launch interface. This ensures only activities in that workspace will be delivered to your mobile app.
 
+## Visual preview
+
+Visual preview mode allows you to easily perform end-to-end QA for Target activities by enrolling and previewing these activities on your device. This mode does not require a specialized testing set up. To get started, set up a URL scheme and generate the preview links. For more information, see [Target mobile preview](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/target-mobile-preview.html).
```

Comment:
- Please replace "QA" with "testing".
- Please make "This mode does not require a specialized testing set up." a "Tip" type note.

Reply:
done